builder/steps/gen-dockerfile/contents/src/data/install-node-version

```diff
@@ -0,0 +1,4 @@
+# Check to see if the the version included in the base runtime satisfies
+# %s, if not then do an npm install of the latest available
```

Comment:
`npm install` occurs later.  If I can switch to using a templating system, this will be more clear.